src/content/en/updates/2017/03/nic57.md

```diff
@@ -0,0 +1,131 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Chrome 57 for developers? Grid based layouts, Media Session API and more.
+
+{# wf_published_on: 2017-03-08 #}
+{# wf_updated_on: 2017-03-08 #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_tags: chrome57,new-in-chrome,css,progressive-web-apps,media,grids #}
+{# wf_featured_snippet: With Chrome 57, you can now use <code>display: grid</code> for grid based layouts, use the <b>media session API</b> to customize the lock screen and notifications with information about the media being played and more. Pete LePage has all the details and how you can use these new developer features in Chrome 57! #}
+
+# New In Chrome 57 {: .page-title }
+
+{% include "web/_shared/contributors/petelepage.html" %}
+
+<div class="clearfix"></div>
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="TODO"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+
+* Chrome 57 adds support for [`display: grid`](#gridlayout) - the new 
+  CSS Grid Layout specification.
+* You can now customize and respond to user input on the lock screen and
+  notifications using the new [Media Session API](#mediasession).
+* And there are plenty [more](#andmore)!
+
+<div class="clearfix"></div>
+
+I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome 57! 
+
+<div class="clearfix"></div>
+
+## CSS Grid Layout {: #gridlayout }
+
+<img src="/web/updates/images/2017/03/nic57/grid-layout.png" class="attempt-right">
+
+Flexbox is a powerful layout tool, it makes many complex layouts possible,
+but it can only do layout in one dimension. Chrome 57 adds support for
+[`display: grid`](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids)
+- the new [CSS Grid Layout specification](https://www.w3.org/TR/css3-grid-layout/),
```

Comment:
The phrase 'the new CSS Grid Layout specification' should be punctuated the same on both ends. Either use commas at both ends or dashes at both ends. If you use dashes, they should be m-dashes. (The entity is &amp;mdash&semi;).